webui/react/src/components/CheckpointModal.tsx

```diff
@@ -0,0 +1,108 @@
+import { Modal } from 'antd';
+import React, { useCallback, useMemo } from 'react';
+
+import Badge, { BadgeType } from 'components/Badge';
+import { CheckpointDetail, CheckpointStorageType, ExperimentConfig, RunState } from 'types';
+import { formatDatetime } from 'utils/date';
+import { humanReadableBytes, humanReadableFloat } from 'utils/string';
+
+import css from './CheckpointModal.module.scss';
+
+interface Props {
+  checkpoint: CheckpointDetail;
+  config: ExperimentConfig;
+  onHide?: () => void;
+  show?: boolean;
+}
+
+const getStorageLocation = (config: ExperimentConfig, checkpoint: CheckpointDetail): string => {
```

Comment:
I think the hostpath is not coming through :thinking: 
![Screen Capture_select-area_20200717111705](https://user-images.githubusercontent.com/12127420/87818376-4aa39f80-c81f-11ea-99c0-34ad9abfd5e4.png)

Reply:
good eye! (not sure why the saying only compliments one eye) I had the storage path absolute path check logic reversed.